training/nc_train.xml

```diff
@@ -0,0 +1,55 @@
+  <AddressString><AddressNumber>27</AddressNumber> <StreetName>OCEAN</StreetName> <StreetName>ISLE</StreetName> <StreetName>WEST</StreetName> <StreetNamePostType>BLVD</StreetNamePostType> <OccupancyIdentifier>#</OccupancyIdentifier> <OccupancyIdentifier>3I</OccupancyIdentifier> <PlaceName>OCEAN</PlaceName> <PlaceName>ISLE</PlaceName> <PlaceName>BEACH</PlaceName> <ZipCode>28469</ZipCode></AddressString>
+  <AddressString><AddressNumber>0</AddressNumber> <StreetNamePreDirectional>E</StreetNamePreDirectional> <StreetName>WARDELL</StreetName> <StreetNamePostType>DR</StreetNamePostType> <OccupancyIdentifier>#</OccupancyIdentifier> <OccupancyType>APT</OccupancyType> <OccupancyIdentifier>3</OccupancyIdentifier> <PlaceName>PEMBROKE</PlaceName> <ZipCode>28372</ZipCode></AddressString>
+  <AddressString><AddressNumber>252</AddressNumber> <StreetName>SLEEPY</StreetName> <StreetName>AVENUE</StreetName> <StreetName>WEST</StreetName> <StreetNamePostType>RD</StreetNamePostType> <PlaceName>WILKESBORO</PlaceName> <ZipCode>28697</ZipCode></AddressString>
+  <AddressString><AddressNumber>160</AddressNumber> <StreetName>JAMES</StreetName> <StreetName>W</StreetName> <StreetName>DAVIS</StreetName> <StreetNamePostType>LN</StreetNamePostType> <PlaceName>SANFORD</PlaceName> <ZipCode>27332</ZipCode></AddressString>
+  <AddressString><AddressNumber>902</AddressNumber> <StreetName>LAKE</StreetName> <StreetName>N</StreetName> <StreetName>SHORE</StreetName> <StreetNamePostType>DR</StreetNamePostType> <PlaceName>MOREHEAD</PlaceName> <PlaceName>CITY</PlaceName> <ZipCode>28557</ZipCode></AddressString>
+  <AddressString><AddressNumber>728</AddressNumber> <StreetName>C</StreetName> <StreetName>AND</StreetName> <StreetName>N</StreetName> <StreetName>SMITH</StreetName> <StreetName>MILL</StreetName> <StreetNamePostType>RD</StreetNamePostType> <PlaceName>STONEVILLE</PlaceName> <ZipCode>27048</ZipCode></AddressString>
+  <AddressString><AddressNumber>820</AddressNumber> <StreetNamePreDirectional>North</StreetNamePreDirectional> <StreetNamePreType>Highway</StreetNamePreType> <StreetName>71</StreetName> <StreetName>Business</StreetName> <PlaceName>Lowell</PlaceName> <StateName>AR</StateName> <ZipCode>72745</ZipCode></AddressString>
+  <AddressString><AddressNumber>100</AddressNumber> <StreetName>GENERAL</StreetName> <StreetName>A</StreetName> <StreetName>S</StreetName> <StreetName>JOHNSON</StreetName> <StreetNamePostType>ST</StreetNamePostType> <PlaceName>STANLEY</PlaceName> <ZipCode>28164</ZipCode></AddressString>
+  <AddressString><AddressNumber>2500</AddressNumber> <StreetNamePreDirectional>S</StreetNamePreDirectional> <NotAddress>AND</NotAddress> <StreetNamePreDirectional>W</StreetNamePreDirectional> <StreetName>FARM</StreetName> <StreetNamePostType>RD</StreetNamePostType> <PlaceName>HICKORY</PlaceName> <ZipCode>28602</ZipCode></AddressString>
```

Comment:
As stated, I cannot confirm its existence, only that multiple voters have indicated that they live on this street. On Zillow's website [here](https://www.zillow.com/homedetails/2320-S-And-W-Farm-Rd-Hickory-NC-28602/82177703_zpid/) it seems like is not an intersection of two streets, however.